data/human/jobs.txt

```diff
@@ -2972,6 +2972,49 @@ mission "Bounty Hunting (Very Tiny)"
 			payment 25000
 			fail
 
+mission "Bounty Hunting (Very Tiny, Local)"
+	name "Wanted corsair entering <system>"
+	description "A small pirate fighter named the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."
```

Comment:
```suggestion
	description "A small pirate fighter named the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```
I feel like this would read a bit better and avoid some confusion.

Reply:
Maybe:
```suggestion
	description "A small pirate fighter named the <npc> has been attacking merchants in nearby star systems, and now it's about to arrive in <system>! Destroy it by <day> for payment (<payment>)."
```

Reply:
That works for me.

Reply:
Here definitely works best, imo, because it far better emphasizes current location than just referencing it.

IF we really want to include system name for -total- player clarity, it would be better to still do "here, in <system>," but that's an awkward presentation that strains in character tone.